src/citra_qt/configuration/config.cpp

```diff
@@ -56,31 +55,31 @@ const std::array<std::array<int, 5>, Settings::NativeAnalog::NumAnalogs> Config:
 // This must be in alphabetical order according to action name as it must have the same order as
 // UISetting::values.shortcuts, which is alphabetically ordered.
 // clang-format off
-const std::array<UISettings::Shortcut, 24> default_hotkeys{
-    {{QStringLiteral("Advance Frame"),            QStringLiteral("Main Window"), {QStringLiteral("\\"), Qt::ApplicationShortcut}},
-     {QStringLiteral("Capture Screenshot"),       QStringLiteral("Main Window"), {QStringLiteral("Ctrl+P"), Qt::ApplicationShortcut}},
-     {QStringLiteral("Continue/Pause Emulation"), QStringLiteral("Main Window"), {QStringLiteral("F4"), Qt::WindowShortcut}},
-     {QStringLiteral("Decrease Speed Limit"),     QStringLiteral("Main Window"), {QStringLiteral("-"), Qt::ApplicationShortcut}},
+const std::array<UISettings::Shortcut, 24> Config::default_hotkeys{
+    {{QStringLiteral("Advance Frame"),            QStringLiteral("Main Window"), {QStringLiteral("\\"),     Qt::ApplicationShortcut}},
```

Comment:
```suggestion
const std::array<UISettings::Shortcut, 24> Config::default_hotkeys{{
    {QStringLiteral("Advance Frame"),            QStringLiteral("Main Window"), {QStringLiteral("\\"),     Qt::ApplicationShortcut}},
```

So that it is similar to what clang-format would do, see `default_analogs` above